src/WebJobs.Script.WebHost/Management/FunctionsSyncManager.cs

```diff
@@ -548,16 +548,26 @@ private async Task<string> GetDurableMajorVersionAsync(JObject hostJson, ScriptJ
             bool isUsingBundles = hostJson != null && hostJson.TryGetValue("extensionBundle", StringComparison.OrdinalIgnoreCase, out _);
             if (isUsingBundles)
             {
-                // TODO: As of 2019-12-12, there are no extension bundles for version 2.x of Durable.
-                // This may change in the future.
-                return "1";
+                // From Functions runtime V4 onwards, only bundles >= V2.x is supported, which implies the app should be using DF V2 or greater.
+                return "2";
```

Comment:
Should we be getting the actual version from the metadata instead of hardcoding this here?

Reply:
@fabiocav: you mean from the extension bundles version range? I thought of that, but worried that parsing the version range via regex might be dangerous and more error prone than it's worth, so I opted for just going with bundles V1 not being supported for Functions V4. Happy to be challenged on this though.

Is there code in the host that can already parse the bundles range? If so, I can re-use that

Reply:
DF major version. Maybe I misunderstood your suggestion

Reply:
That would be available (as the actual version) with the extensions resolved by the the bundle. I don't think we need to actually block on that, but lookin at the actual extension type would give you the exact major version. We already have code in the script startup type locator to parse the extensions once you get what is resolved by the bundles. 

Again, not a blocker.

Reply:
That would be a better approach, so I'd be happy to refine this logic later. However, as it's not a blocker, and given that Host v4 already fails fast when bundles V1 is loaded (just validated this locally), I think I'd rather keep this change as-is for simplicity and expediency's sake.

If all goes according to plan, we should be removing this DF specific logic from the Host anyways, so hopefully this is a short term solution for a codepath that is soon to be removed :)